circle.yml

```diff
@@ -21,7 +21,7 @@ dependencies:
 
 test:
   pre:
-    - cd $CPATH && go run cli/main.go --migrate
+    - cd $CPATH && go run cli/main.go --migrate -config config.json
```

Comment:
This is not directly related to this PR, but why we use two dashes for `migrate` here? `flag` package uses one dash by default.

Reply:
Well technically it should be 2 dashes with a long option as the posix standard is a single dash when its a single letter name and a double dash for a word.

https://www.gnu.org/software/libc/manual/html_node/Argument-Syntax.html

Reply:
I know but go has its own way. :)

Reply:
I would say in this case the wrong way 😝 why break with tradition? I actually prefer the flags packages that impose the posix standard on go flags! (for example pflags or viper). We had a ticket about this too somewhere as someone asked why we don't follow the standard. In reality as the flags package lets us specify either I think we are all good for now 😄